src/eh1/delay.rs

```diff
@@ -1,5 +1,5 @@
 //! Delay mock implementations, implementing both sync and async
-//! [`DelayNs`](https://docs.rs/embedded-hal/latest/embedded_hal/delay/trait.DelayNs.html)
+//! [`BlockingDelay`](https://docs.rs/embedded-hal/latest/embedded_hal/delay/trait.BlockingDelay.html)
```

Comment:
Was this change done by mistake? `BlockingDelay` is not a trait that exists in e-h.

I suspect this was done accidentally as part of an automated rename.

Reply:
Yup, accidental rename. Fixed.